### PR TITLE
Simplify command to access Dashboard

### PR DESCRIPTION
**Pull request by johscheuer**
Signed-off-by: Johannes M. Scheuermann <joh.scheuer@gmail.com>

# Changes

Simplify command to access Dashboard and do some Readme clean up with markdown lint.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,27 +42,32 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
...
 You can access the web UI at `http://localhost:9097/`
 
 ## Uninstalling the Dashboard on Minishift
+
 The Dashboard can be uninstalled on Minishift by running the command `./minishift-delete-dashboard.sh` Use `-n {NAMESPACE}` on the end of the command if installed into a namespace other than `tekton-pipelines`
 
 ## Accessing the Dashboard
-The Dashboard can be accessed through its ClusterIP Service by running `kubectl proxy`. Assuming tekton-pipelines is the install namespace for the dashboard, you can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`
+
+The Dashboard can be accessed through its ClusterIP Service by running `kubectl proxy`. Assuming tekton-pipelines is the install namespace for the dashboard, you can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`. An alternativ way to access the Dashboard is using `kubeectl port-forward` e.g. if you installed the Tekton Dashboad into the `tekton-pipelines` namespace (which is the deafult) you can access the Dashboard with `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097`
```
Comment (vtereso) [inline README.md]: `kubeectl` -> `kubectl` 
`alternativ` -> `alternative`

Comment (johscheuer) [reply README.md]: Thanks for the catch!

**Comment by johscheuer** [comment]
From my point of view the `kubectl port-forward` is cleaner then using `kubectl proxy` since it actually describes what I want instead of setting up the proxy and then using the rather long url.

**Comment by steveodonovan** [comment]
+1 on having both (thanks @johscheuer )

**Comment by a-roberts** [comment]
@johscheuer yep looks good to me too! Could you please squash this to be one commit and then let's get it in? We don't have a "squash and merge" button 🏳️

**Comment by johscheuer** [comment]
Squased

**Comment by a-roberts** [comment]
Thanks, I've raised an issue already for the flaky test failure you're seeing

/retest

**Pull request merged**